tests/support/activemq/message_details.py

```diff
@@ -1,26 +1,33 @@
 import json
+
 from time import sleep
 
 import requests
 
+from django.conf import settings
 from parsel import Selector
+
 from tests.support.dtos import MessageStatus
 
 
-def retrieve_message_published(destination_name, host="localhost") -> MessageStatus:
+def retrieve_message_published(
+    destination_name: str,
+    host: str = settings.STOMP_SERVER_HOST,
+    port: str = settings.STOMP_SERVER_INTERFACE_PORT,
```

Comment:
**suggestion:** Same thing here. Since these params have a default value, the type could be `Optional`.